sdk/Sdk/Sdk.csproj

```diff
@@ -2,7 +2,6 @@
 
   <PropertyGroup>
     <MinorProductVersion>9</MinorProductVersion>
-    <VersionSuffix>-preview2</VersionSuffix>
```

Comment:
Synced with @liliankasem The attributes needed for SDK binding changes were rolled back from main. So without those, the code path won't be activated.